src/omotes_simulator_core/entities/network_controller.py

```diff
@@ -218,7 +251,7 @@ def _set_producers_based_on_priority(self, time: datetime.datetime) -> dict:
         :return: dict with the key the asset id and the value a dict with the set points for the
         producers.
         """
-        producers = self._set_producers_to_max()
+        producers = self._get_basic_producers()
         total_demand = self.get_total_demand(time) + self.get_total_charge_storage()
         if total_demand > self.get_total_supply():
             raise ValueError(
```

Comment:
General question: why raise a ValueError? Should this not be a warning but continue the simulation with capped demands?

Reply:
Basically you can never come here, since before this routine is called it is already checked if this happens and if so then the demand is indeed capped after giving a warning. So I propose to leave this.